docs/rfc/active/data-permissions/README.md

```diff
@@ -0,0 +1,138 @@
+- Start Date: 2022-04-19
+- RFC PR: #4694
+- Discussion Issue: 
+- Implementation PR(s): 
+
+# Modeling Data Permissions
+
+## Summary
+
+Many datahub sources provide access control mechanisms to manage user access to assets.
+Datahub should provide functionality for viewing these permissions.
+
+## Motivation
+
+Within many sql systems, the data control language provides commands like GRANT, DENY and REVOKE to manage control to datasets.
+
+Being able to view grants is the first step toward supporting larger integrations with dataset permissions. [See Here](https://feature-requests.datahubproject.io/b/feedback/p/access-control-on-datasets-bi-assets)
+
+Of the datahub integration sources listed, the following have some form of data access control management:
+
+- Athena (via IAM)
+- BigQuery
+- Clickhouse
+- Druid
+- Elastic Search
+- Glue
+- Hive (via Apache Ranger)
+- Kafka
+- Looker
+- MariaDB
+- Metabase
+- MongoDB
+- MicrosoftSQLServer
+- MySQL
+- Oracle
+- PostgreSQL
+- Redash
+- Redshift
+- S3 (via IAM)
+- Sagemaker
+- Snowflake
+- Superset
+- Tableau
+- Trino
+
+## Requirements
+
+The permissions aspect must be:
+
+- Appliable to datasets, containers, dashboards
+- Support ingesting and viewing structured access control (GRANT {permission} ON {object} TO {user or role}
+- Support ingesting and viewing unstructured access control (IAM Policy Documents, Custom Policies from other datasets)
+-
```

Comment:
I'd at least consider fine-grained permissions, column filter/mask, and row level security data permissions when designing the data models for permissions. Only certain datastores would support this level, however it is relatively common in enterprise environments and present in quite a few of the target sources listed above.